vulyk/assets.py

```diff
@@ -1,9 +1,13 @@
-# coding=utf-8
+# -*- coding: utf-8 -*-
+
 from flask.ext.assets import Environment, Bundle
 
 
 def init(app):
+    """ Bundle projects assets """
+
```

Comment:
How should plugin-related assets be collected now after you remove the code from here?
I supposed that you just move it into plugin, add one more filename (for each js and css) into global settings (or even local if we separate it completely) and add them to the queue.